### PR TITLE
Typos fixed, WA sandbox url fixed and content-type header added

### PR DESCRIPTION
**Pull request by lucvanderzandt**
Fixed typos in the feature name and url of the WA sandbox and added content-type header to all requests. Furthermore, I added needed dependencies to the project.

Please tag the project with tag 1.3.3.

### CODE REVIEW

File: lib/src/general/model/base_service.dart [hunk 1]
```diff
@@ -31,9 +31,9 @@ abstract class BaseService {
   /// Constructor.
   BaseService(this._accessKey, {int timeout, List<String> features})
       : _timeout = (timeout == null) ? 5000 : timeout {
-    if (features?.contains('ENDABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX') ??
+    if (features?.contains('ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX') ??
         false) {
```
Comment (Multimo) [inline lib/src/general/model/base_service.dart:35]: Dart has some really nice features :) Having no truthy values feels like its for the best.

**Review by Multimo — APPROVED** [review]
Looks good to me! 👍

**Comment by Multimo** [comment]
@lucvanderzandt merging this since it seems like its ready to go in. If this was a mistake let me know I can do a revert

**Comment by Multimo** [comment]
@lucvanderzandt merging this since it seems like its ready to go in. If this was a mistake let me know I can do a revert

**Comment by lucvanderzandt** [comment]
Thank you! Could you also [tag a release](https://github.com/messagebird/dart-rest-api/issues/2) again?

If you so desire, I could also help maintaining this repository. I don't know if GitHub supports maintaining a repository without being part of the organisation, but my offer stands ;)

**Pull request merged**